### ISSUE TITLE
Make a Tag page for listing shows

### ISSUE THREAD

**Issue by baxgas** [mmmnmnm/lahmacun#224]
https://www.lahmacun.hu/tag/music/
Have a better overview with 2-3-4 (?) covers and text in row, maybe Masonry.js layout. 
Infos: cover image, title

**Comment by baxgas** [comment]
Also will need to put the tags list somewhere on the Show page, maybe under the Cover image?
Eg.:
![image](https://user-images.githubusercontent.com/36408851/79265923-21c5e400-7e97-11ea-9b1f-e0b526cff3e5.png)

**Comment by gammaw** [comment]
Adjust template but don't tags/categories at this point

**Comment by baxgas** [comment]
Made the Shows page on the basis of the menu-look (3 in a row), and also copied the same to the Tags page if needed in the future. The front-end menu Show item will need to have the link coded in from wordpress to point to it

**Comment by gammaw** [comment]
Had to revert it: https://github.com/mmmnmnm/lahmacun/pull/233

**Comment by gammaw** [comment]
@baxgas the style looks good but is it possible that News category Posts are also listed? Maybe I'm doing something wrong.

**Comment by gammaw** [comment]
Btw, how does Wordpress find `/shows`? Is it some kind of automated routing?

**Comment by baxgas** [comment]
> @baxgas the style looks good but is it possible that News category Posts are also listed? Maybe I'm doing something wrong.

Maybe some News posts also got some shows class by mistake? In this page everything gets listed which is under the Shows category
Is the "order by date" OK for this, or should it be alphabetically?

**Comment by gammaw** [comment]
> Maybe some News posts also got some shows class by mistake? In this page everything gets listed which is under the Shows category

I know. The categories look ok (either Show or News), still they were both shown to me. Can you try on dev? (I guess it's now on master but do check.). 

> Is the "order by date" OK for this, or should it be alphabetically?

Since the menu is alphabetical, I'd do the same here.

**Comment by baxgas** [comment]
Yes, it seems some posts got there by mistake on dev
![image](https://user-images.githubusercontent.com/36408851/81003124-a7133780-8e4a-11ea-839d-7a6db2650ddd.png)
I'll fix it. Also the commit with alphabetical ordering

**Comment by gammaw** [comment]
> Yes, it seems some posts got there by mistake on dev

What do you mean? Szilveszteri Élő Show was an actual show. I see no false categories on the screenshot. Didn't you see News items listed by the new template?

**Comment by gammaw** [comment]
Strange, it looks perfect now. Maybe I was doing something wrong. Sorry. 

> Btw, how does Wordpress find /shows? Is it some kind of automated routing?

@baxgas can you help me answering this? I know it's a rookie question but how is `/category/shows` equal to `/shows`?

**Comment by baxgas** [comment]
@gammaw 
Yes, the wordpress has automatic permalink rewriting for routing. The pattern can be set in settings
The /categories is the default way, but on our live site the SEO plugin removes it (think it is better ranking like that)

**Comment by gammaw** [comment]
@baxgas merged it to master and it's on live but I think we can improve it further: 
- How about having three columns? 
- How about showing more text in the thumbnail? 

Also, should we get rid of the shows list view when one hover over the shows menu? 

![image](https://user-images.githubusercontent.com/28865986/85801950-33075880-b744-11ea-9e8a-5cdbe0532fa9.png)

**Comment by baxgas** [comment]
@gammaw yes, you are right, we don't need the menu with the new Shows page. 
Unfortunately there was some mess with the merge you made and many stuff got lost :-( 
I fixed it in this commit: https://github.com/mmmnmnm/lahmacun/commit/52efb1c03ed381e0962398f255d99951620d37f2 can go live like that (I think it is not a good idea to have so long delays, because of the changes in the meantime and merge-conflicts)

**Comment by gammaw** [comment]
@baxgas I see it's 3-column now, which is good. How can we remove the content area? Isn't it code? 

For your commits: I see hundreds of newly added files. What are those? 

![image](https://user-images.githubusercontent.com/28865986/85948315-8a7a1400-b950-11ea-8eb5-038bbc56ac0d.png)

**Comment by baxgas** [comment]
@gammaw – the added files were merged from the master branch, I believe Arcsi is already in there?

> How can we remove the content area? Isn't it code?
What do you mean with "Content Area"?

**Comment by gammaw** [comment]
@baxgas 

> the added files were merged from the master branch, I believe Arcsi is already in there?

I don't think so. This is how the master branch looks like: https://github.com/mmmnmnm/lahmacun/tree/master/wp/wp-content/themes/lahma_maker No trace of the `arcsiplayer` folder. 

Maybe you did a merge from your local master where you already added (and committed) those files. This is the commit history on the branch: https://github.com/mmmnmnm/lahmacun/commits/%23224-Shows-categories/wp/wp-content/themes/lahma_maker

> What do you mean with "Content Area"?

This:

![image](https://user-images.githubusercontent.com/28865986/85948569-e1341d80-b951-11ea-9cde-44a64a549639.png)

**Comment by baxgas** [comment]
Oh, scheise, I'll delete the player files, sorry

If the old menu is obsolete (this branch is up on prod), the Shows menu item can be replaced with a simple one-level link to the Shows page

**Comment by baxgas** [comment]
@gammaw ok, removed the obsolete code for the menu hover and cleanup code. when on prod, need to make the Shows menu item as a one-level-depth link to the shows Category page

**Comment by gammaw** [comment]
@baxgas look goods! I see a new tag.php file. What's that for? I don't see it would take effect.

**Comment by baxgas** [comment]
@gammaw  the tag is just listing the tags we wanted presenting at some point (eg. music, talk-show, etc.), so it is using the same styling as the Shows and is not broken as by default

**Comment by gammaw** [comment]
@baxgas looks good, it's on live now. 

I think we could do some improvement right away. There are some strange results because of the same text structure on the sub page (e.g., picture). What if we check if the excerpt field of the show is empty and if it is _not_ then we use that for the text on the shows page? In this way, we don't always need to fill the excerpt field, only if the result doesn't look good. 

![image](https://user-images.githubusercontent.com/28865986/86539737-4910d800-beff-11ea-8485-1cb7e5c56a02.png)

**Comment by baxgas** [comment]
@gammaw Thanks!
The exceprts can be custom made per post, not to have the automatic generated. I checked AZVLM and fixed it, could be also made for all the Shows, and then to even turn off the automatic excerpt truncation

**Comment by baxgas** [comment]
@gammaw I increased the word length for truncation, should work fine, but the custom excerpt is a much better solution, since usually we have the showtime/archive info block at the end which is not needed here
